backend/stac_format.py

```diff
@@ -26,4 +28,12 @@
 STAC_TYPE_COLLECTION = "Collection"
 STAC_TYPE_ITEM = "Feature"
 STAC_TYPE_KEY = "type"
+STAC_UPDATED_KEY = "updated"
+STAC_VERSION = "1.0.0"
 STAC_VERSION_KEY = "stac_version"
+
+LATEST_LINZ_STAC_VERSION = "0.0.3"
```

Comment:
We could get this by version ordering the directories in the submodule. But it would be a fairly expensive operation, and hopefully Dependabot updating the submodule should trigger `should_treat_linz_example_json_files_as_valid` to fail if there are any semantic changes to the repo.

Reply:
@billgeo Are we definitely going to need to support more than one version of the LINZ STAC extension?

Reply:
In the near future, I can't think of a case where we'd need to support older versions. Seems like this is all about compliance and ensuring all metadata is consistent with the latest version.